VCID/src/main/java/com/aliucord/plugins/VCID.java

```diff
@@ -34,13 +34,13 @@ public void start(Context context) throws Throwable {
         // add the patch
         patcher.patch(WidgetChannelsListAdapter.ItemChannelVoice.class.getDeclaredMethod("onConfigure", int.class, ChannelListItem.class), new Hook(callFrame -> {
             var channel = (ChannelListItemVoiceChannel) callFrame.args[1];
-            if (PermissionUtils.can(16, channel.component1().h())) return;
+            if (PermissionUtils.can(16, channel.component1().g())) return;
```

Comment:
wrong getter, it's supposed to get the channel id not flags

Reply:
ah ok ty for correcting !

Reply:
are you going to push a fix?

Reply:
oh sorry i thohught it was accepted by default